src/notation/view/widgets/editstringdata.ui

```diff
@@ -27,20 +19,23 @@
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
        <item>
         <widget class="QTableWidget" name="stringList">
-         <attribute name="horizontalHeaderDefaultSectionSize">
-          <number>50</number>
-         </attribute>
-         <attribute name="horizontalHeaderStretchLastSection">
-          <bool>true</bool>
-         </attribute>
+         <property name="sizePolicy">
+          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+           <horstretch>0</horstretch>
+           <verstretch>0</verstretch>
+          </sizepolicy>
+         </property>
          <column>
           <property name="text">
-           <string>Always open</string>
+           <string notr="true">Always open</string>
+          </property>
+          <property name="toolTip">
+           <string>On tablature staves, fret positions other than &quot;0&quot; cannot be entered on strings marked 'always open'. Useful for instruments with strings that are not on the fretboard.</string>
```

Comment:
Missing `<b>Always open</b><br>` at the beginning.

You can use curly quotes here too: `“0” ‘always open’`. But wait for a reply to https://github.com/musescore/MuseScore/issues/19362#issuecomment-1715401132.

Reply:
Curly quotes around numbers lead to compiler warnings:
"decimal digit terminates octal escape sequence (C4125)"

Reply:
Strange. I see people complaining about that online, but I don't see it myself with MSVC.

Did you enable extra warnings? (How?)

Reply:
No extra warnings enabled. Just the same build I use since long. VS 2022

Reply:
If I do it via C++ code (i.e. in editstringdata.cpp, to have it on every string line) I get
The file contains a character starting at offset 0xb14 that is illegal in the current source character set (codepage 65001). (C4828)

Reply:
Ah, seems I've overcome that now.

Reply:
I guess adding the `qtrc()` did the trick (of marking the strings being Unicode?)